flink-filesystems/flink-s3-fs-base/pom.xml

```diff
@@ -231,6 +231,33 @@ under the License.
 					<skip>true</skip>
 				</configuration>
 			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>shade-flink</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<filters>
+								<filter>
+									<artifact>com.amazonaws:aws-java-sdk-s3</artifact>
```

Comment:
this should no longer be necessary since `flink-s3-fs-base` no longer bundles this dependency

Reply:
Good catch. Not sure why I thought it was necessary.